core/src/main/java/org/testcontainers/containers/image/pull/policy/PullPolicy.java

```diff
@@ -0,0 +1,38 @@
+package org.testcontainers.containers.image.pull.policy;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Convenience class with logic for building common {@link ImagePullPolicy} instances.
+ *
+ */
+public class PullPolicy {
```

Comment:
Again a small thing, but this class needs to not be instantiable.

Reply:
That's a good point, should the same apply to org.testcontainers.containers.wait.strategy.Wait ?

Reply:
Well spotted! Yes, would you mind altering that class too as part of this PR?

The Lombok `@UtilityClass` annotation is what we've used in other places.